src/app/job-routing/my-history/my-history.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { myApplication } from '../my-application/my-application.module';
+
+@Component({
+  selector: 'app-my-history',
+  templateUrl: './my-history.component.html',
+  styleUrls: ['./my-history.component.css']
+})
+export class MyHistoryComponent {
```

Comment:
i really do not agree to this data structure. let your DS be much more readable and clear

Reply:
you are creating something like this... --> 
[{
"image": "./assets/image.jpg",
"companyName": "ABC company",
"JobLevel: "Senior Level Engineer",
"country": "UAE",
"proposedSalary": 10000,
"skillSet": ['figma','figma','figma','figma','figma',],
"applicationDate": "Applied 25 days ago"
},
{
"image": "./assets/image.jpg",
"companyName": "ABC company",
"JobLevel: "Senior Level Engineer",
"country": "UAE",
"proposedSalary": 10000,
"skillSet": ['figma','figma','figma','figma','figma',],
"applicationDate": "Applied 25 days ago"
},
{
"image": "./assets/image.jpg",
"companyName": "ABC company",
"JobLevel: "Senior Level Engineer",
"country": "UAE",
"proposedSalary": 10000,
"skillSet": ['figma','figma','figma','figma','figma',],
"applicationDate": "Applied 25 days ago"
},
{
"image": "./assets/image.jpg",
"companyName": "ABC company",
"JobLevel: "Senior Level Engineer",
"country": "UAE",
"proposedSalary": 10000,
"skillSet": ['figma','figma','figma','figma','figma',],
"applicationDate": "Applied 25 days ago"
},]